src/utils/tokens.ts

```diff
@@ -1,6 +1,9 @@
 import { Token } from '@uniswap/sdk-core'
-import { NetworkInfo, PolygonNetworkInfo } from 'constants/networks'
-import { WETH_ADDRESSES } from '../constants/index'
+import { CeloNetworkInfo, NetworkInfo, PolygonNetworkInfo } from 'constants/networks'
+import { WETH_ADDRESSES } from '../constants'
+
+const CELO_ADDRESS = '0x471EcE3750Da237f93B8E339c536989b8978a438'
```

Comment:
Do not redefine these constants. Export the token addresses from src/constants/index.ts L8-10 and use them in this file.